cockatrice/src/filtertree.cpp

```diff
@@ -295,6 +295,27 @@ bool FilterItem::acceptRarity(const CardInfoPtr info) const
     return false;
 }
 
+bool FilterItem::relationCheck(int cardInfo) const
+{
+    bool result, conversion;
+
+    result = (cardInfo == term.toInt(&conversion));
+    if (!conversion)
+    {
+        int termInt = term.mid(1).toInt();
+        if (term.startsWith('<')) {
```

Comment:
Also thinking just a bit further... can this possibly be simplified to

`result = (cardInfo term.left(1) termInt)` or something along those lines?

Reply:
It would be really nice, but I'm not sure how could that work in C++. Can Qt somehow convert a char into a logical operator?